common/djangoapps/enrollment/views.py

```diff
@@ -469,6 +472,14 @@ def get(self, request):
                     ).format(username=username)
                 }
             )
+        if username == request.user.username or self.has_api_key_permissions(request):
+            return Response(enrollment_data)
+        filtered_data = []
+        for enrollment in enrollment_data:
```

Comment:
@dsjen I'm currently using `student.auth.user_has_role()`, which at least covers [instructors and beta testers](https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/auth.py#L46-L48), but not organizational users.  I would have used `courseware.access.has_access()` but can't, since it is in an LMS library which I can't import from `common/`.

I'm wondering why there are multiple sligtly different implementations of the access rules anyway. :)